### PR TITLE
Add reporting of cli args of caller that starts svc

### PR DESCRIPTION
**Pull request by daved**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1030" title="DX-1030" target="_blank">BugDX-1030</a>  state-svc foreground []: state-svc errored out: Could not start service: Failed to start server: flisten in use
  </td></table>
  <br />

### CODE REVIEW

File: cmd/state-svc/main.go [hunk 1]
```diff
@@ -151,30 +153,41 @@ func run(cfg *config.Instance) error {
 			cmdForeground,
 			"Starting the ActiveState Service",
 			"Start the ActiveState Service (Foreground)",
-			p, nil, nil,
+			p, nil,
+			[]*captain.Argument{
+				{
+					Name:        "Arg text",
+					Description: "Argument text of caller",
```
Comment (Naatan) [inline cmd/state-svc/main.go:160]: The description should answer the question "Why should I provide this?".

**Review by Naatan — CHANGES_REQUESTED** [review]
Can we also improve the error of `flisten in use` so that it conveys that this error is partly expected? Otherwise this will continue to look like something that needs fixing.

**Review by Naatan — CHANGES_REQUESTED** [review]
You missed this comment I think: https://github.com/ActiveState/cli/pull/1996#pullrequestreview-1065554573

**Comment by daved** [comment]
@Naatan Sorry about missing that. I also added a test.

File: cmd/state-svc/service.go [hunk 2]
```diff
@@ -58,6 +58,9 @@ func (s *service) Start() error {
 	s.ipcSrv = ipc.NewServer(s.ctx, spath, reqHandlers...)
 	err = s.ipcSrv.Start()
 	if err != nil {
+		if errors.Is(err, ipc.ErrInUse) {
+			return errors.New("An existing server instance appears to be in use")
```
Comment (Naatan) [inline cmd/state-svc/service.go:62]: This should still wrap imo.

**Review by Naatan — CHANGES_REQUESTED** [review]

**Review by Naatan — APPROVED** [review]

**Pull request merged**